### ISSUE TITLE
Complaining when dot is missing in examples

### ISSUE THREAD

**Issue by mitar** [tetafro/godot#29]
Go supports examples in tests which end with comment like:

```
// Output: true
```

To document the expected output of the example code. Godot complains here about a missing dot. I think it should not complain about output comments in examples.

**Comment by tetafro** [comment]
Fixed in [v1.4.15](https://github.com/tetafro/godot/releases/tag/v1.4.15)

**Comment by webstradev** [comment]
So just wanted to raise another similar issue thought it would be easier to reopen this one.
This fix only works for single line output, but a lot of example tests often have multiple lines and the pattern would be something like this:
```go
// Output:
// true
// false
```

Also, go supports unordered output like this:

```go
// Unordered Output:
// false
// true
```

**Comment by tetafro** [comment]
Fixed in [v1.4.17](https://github.com/tetafro/godot/releases/tag/v1.4.17)
